TICoordinatorKit/Classes/Routable/Modal/ModalRoutable.swift

```diff
@@ -25,9 +25,12 @@ import UIKit
 public protocol ModalRoutable: Presentable {
 
     typealias NavigationControllerFactory = (UIViewController) -> UINavigationController
+    typealias ConfigurationClosure = (UIViewController) -> ()
     
     func present(_ module: Presentable?)
     func present(_ module: Presentable?, animated: Bool)
+    func present(_ module: Presentable?, configurationClosure: ConfigurationClosure?)
```

Comment:
this method looks superfluous
there 2 "simple" methods and 1 complex

Reply:
this applies to the case of 'animated = true', because almost all the time the animation is true. just an auxiliary method which simplifies the call of method